configs/3dssd/README.md

```diff
@@ -29,6 +29,6 @@ Some settings in our implementation are different from the [official implementat
 
 |  Backbone   |Class| Lr schd | Mem (GB) | Inf time (fps) | mAP |Download |
 | :---------: | :-----: | :------: | :------------: | :----: |:----: | :------: |
-|    [PointNet2SAMSG](./3dssd_kitti-3d-car.py)| Car |72e|4.7||78.69(81.27)<sup>1</sup>|[model](https://download.openmmlab.com/mmdetection3d/v0.1.0_models/3dssd/3dssd_kitti-3d-car_20210602_124438-b4276f56.pth) &#124; [log](https://download.openmmlab.com/mmdetection3d/v0.1.0_models/3dssd/3dssd_kitti-3d-car_20210602_124438.log.json)|
+|    [PointNet2SAMSG](./3dssd_4x4_kitti-3d-car.py)| Car |72e|4.7||78.69(81.27)<sup>1</sup>|[model](https://download.openmmlab.com/mmdetection3d/v0.1.0_models/3dssd/3dssd_kitti-3d-car_20210602_124438-b4276f56.pth) &#124; [log](https://download.openmmlab.com/mmdetection3d/v0.1.0_models/3dssd/3dssd_kitti-3d-car_20210602_124438.log.json)|
```

Comment:
Is it necessary to modify the path name too?

Reply:
We can keep the download link